<launch>
  <group ns="valve_planner">
    <node pkg="hubo_planner" type="drchubo_planner_interface.py" name="drchubo_planner_interface" output="screen"/>
    <param name="robot_model" value="$(find drchubo-v2)/robots/drchubo-v2.robot.xml"/>
    <param name="tiny_wheel_model" value="$(find wpi_planning_utilities)/../models/driving_wheel_tiny.robot.xml"/>
    <param name="read_joint_states" type="bool" value="false"/>
  </group>
</launch>